src/screenshot.js

```diff
@@ -1,13 +1,14 @@
 const handlebars = require('handlebars')
 
 module.exports = {
-  makeScreenshot: async function(page, {
+  makeScreenshot: async function (page, {
     output,
     type,
     quality,
     encoding,
     content,
     html,
+    beforeScreenshot = () => { },
```

Comment:
This line set a default value which works fine only is nothing is passed to the function. If anything else is passed to this argument it won't works (string, numbers, null, ...). I think we should check if beforeScreenshot is a `function` before calling it. What do you think of it?

Reply:
I agree and will implement a typecheck for `beforeScreenshot`